docker-run.sh

```diff
@@ -2,14 +2,9 @@
 
 set -e
 
-BASE_URL=${BASE_URL:-/}
 NGINX_ROOT=/usr/share/nginx/html
 INDEX_FILE=$NGINX_ROOT/index.html
-NGINX_CONF=/etc/nginx/nginx.conf
-
-if [[ "${BASE_URL}" != "/" ]]; then
-  sed -i "s|location / {|location $BASE_URL {|g" $NGINX_CONF
-fi
+NGINX_CONF=/etc/nginx/conf.d/default.conf
```

Comment:
TECHNICAL NOTE EXPLAINING THE CHANGE

![image](https://user-images.githubusercontent.com/193286/226854766-f5601d7e-11e9-439b-a5ff-ea767f8a3163.png)

Reply:
- We place our configuration into template folder: https://github.com/swagger-api/swagger-editor/pull/3958/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R8
- It will be [modified by the nginx script](https://github.com/nginxinc/docker-nginx/blob/master/mainline/alpine-slim/20-envsubst-on-templates.sh) and moved to the `/etc/nginx/conf.d/<file name>` in our case`/etc/nginx/conf.d/default.conf`, here is the [docu](https://github.com/docker-library/docs/tree/master/nginx#using-environment-variables-in-nginx-configuration-new-in-119) --> Thats why path was changed.

Reply:
Yep, this comment was for reference purposes only. No action on your side was needed.

Reply:
Yep, this comment was for reference purposes only. No action on your side was needed.